cmd/test/k8sValidVersions.go

```diff
@@ -0,0 +1,111 @@
+package test
+
```

Comment:
also, please update the branch from main
![Screen Shot 2021-12-22 at 14 15 17](https://user-images.githubusercontent.com/39004075/147091387-2cd745d2-27a5-4c48-a8f5-1c503a0951b0.png)

Reply:
okay.

Reply:
done